config/manager/kustomization.yaml

```diff
@@ -5,6 +5,12 @@ generatorOptions:
   disableNameSuffixHash: true
 
 configMapGenerator:
-- name: manager-config
-  files:
+- files:
```

Comment:
I think you commited this file by mistake, as this file is modified by kustomize when you deploy the operator

Reply:
No mistake, I'm aware that it's modified by kustomize when the operator is deployed. However as the code stands this is what would be in the file.

If changes are made this file will still be modified. However if one were to just checkout the code and `make deploy` this file would not be modified. I feel that's a reasonable expectation if one doesn't modify the code otherwise.

Reply:
Ok, seems to work, I thought this file wouldn't be modified once kustomize had changed it. :)